web-frontend/src/project-page/ProjectWidget.svelte

```diff
@@ -209,64 +251,73 @@
     <!-- Keywords -->
     {#if $projectMetadata?.project.keywords}
       <div class=label>Keywords</div>
-      <span class="keyword">{$projectMetadata?.project.keywords.map(t => {return getText(t)}).join(", ")}</span>
-    {:else if isTestEnvironment}
+      <span class="keyword">{$projectMetadata?.project.keywords.map(t => {
+        return getText(t)
+      }).join(", ")}</span>
+    {:else if $isTestEnvironment}
       <div class=label>Keywords</div>
       <div class=warning>keywords missing</div>
     {/if}
   </div>
 
-  {:else if isTestEnvironment}
-    <div class=warning>Project not available</div>
+{:else if $isTestEnvironment}
+  <div class=warning>Project not available</div>
 {/if}
 
 <style>
-  a {
-    display: block;
-    color: var(--lead-colour);
-  }
-  /* a::after {
-    display: block;
-    content: ' ';
-    background-image: url('assets/icon/chevron.svg');
-    background-size: 28px 28px;
-    height: 28px;
-    width: 28px;
-  } */
-  .keyword {
-    padding: 0;
-    
-  }
-  .label, .data {
-    margin: 5px 0;
-  }
-  .label {
-    padding: 10px 0 0;
-  }
-  button {
-    border: none;
-    background-color: inherit;
-    padding: 0;
-    position: relative;
-    top: 10px;
-    color: var(--lead-colour);
-    z-index: 0;
-    margin: -1rem 0 0.25rem;
-  }
-  .icon:hover {
-    color: var(--dasch-hover);
-  }
-  .widget {
-    border: 1px solid #cdcdcd;
-    border-radius: 3px;
-    background-color: var(--dasch-grey-3);
-    margin: 15px 0;
-    padding: 0 10px 10px;
-    box-shadow: var(--shadow-1);
-  }
-  .chevron {
-    padding: 0;
-    height: 100%;
-    vertical-align: text-bottom;
-  }
+    a {
```

Comment:
Another formatting detail, entire file had 2 spaces indention, IMO it should remain like that.

Reply:
I like that maybe you create a followup issue to improve the svelte setup. I would like that.